File: day_i/part1/part1.js
const input = ``

console.log(input.split("\n")
    .map(x => 
        parseInt(
            (
                Array(...x)
                    .filter(x => x >= '0' && x <= '9')[0]
            ) + 
            Array(...x)
                .filter(x => x >= '0' && x <= '9').reduce((x, y) => y))
    ).reduce((x, y) => x + y)
)

File: day_i/part1/part1.py
print(
    sum(
        [
            (
                lambda x: int(
                    f"{list(filter(lambda x: x>='0' and x<='9',x))[0]}{list(filter(lambda x: x>='0' and x<='9',x))[-1]}"
                )
            )(i)
            for i in open("input.txt", "r").readlines()
        ]
    )
)
